src/components/Button.jsx

```diff
@@ -0,0 +1,49 @@
+const Button = ({ value }) => {
+  let style =
+    'h-12 w-1/5 rounded bg-themeOne-80 text-3xl text-themeOne-100 shadow-ash hover:bg-themeOne-90 active:translate-y-1';
+
+  switch (value) {
+    case 'DEL':
+      style =
+        'delete-btn h-12 w-1/5 rounded bg-themeOne-120 text-2xl font-bold text-themeOne-white shadow-darkblue hover:bg-themeOne-130 active:translate-y-1';
+      break;
+    case 'RESET':
+      style =
+        ' h-12 w-[48%] rounded bg-themeOne-150 text-2xl font-bold text-themeOne-white shadow-darkblue hover:bg-themeOne-160 active:translate-y-1';
+      break;
+    case '=':
+      style =
+        ' h-12 w-[48%] rounded bg-themeOne-180 text-2xl font-bold text-themeOne-white shadow-red hover:bg-themeOne-190 active:translate-y-1';
+      break;
```

Comment:
tailwind classes like `rounded`, `h-12`, `active:translate-y-1` should not be repeated, let's add those to the button className directly